mon-pix/tests/unit/services/url-test.js

```diff
@@ -32,46 +36,79 @@ describe('Unit | Service | locale', function() {
 
   describe('#homeUrl', function() {
 
-    it('should get default home url when is defined', function() {
-      // given
-      const service = this.owner.lookup('service:url');
-      service.definedHomeUrl = 'pix.test.fr';
-
-      // when
-      const homeUrl = service.homeUrl;
-
-      // then
-      expect(homeUrl).to.equal(service.definedHomeUrl);
+    [
+      { environment: 'development', isRA: 'false' },
+      { environment: 'production', isRA: 'true' },
+    ].forEach((testCase) => {
+      context(`when environnement=‘${testCase.environment}‘ and isRA=${testCase.isRA}`, function() {
+        const defaultEnvironment = ENV.environment;
+        const defaultIsReviewApp = ENV.APP.REVIEW_APP;
+
+        beforeEach(function() {
+          ENV.environment = testCase.environment;
+          ENV.APP.REVIEW_APP = testCase.isRA;
+        });
+
+        afterEach(function() {
+          ENV.environment = defaultEnvironment;
+          ENV.APP.REVIEW_APP = defaultIsReviewApp;
+        });
+
+        it('should get default home url', function() {
+          // given
+          const service = this.owner.lookup('service:url');
+          service.definedHomeUrl = 'pix.test.fr';
+
+          // when
+          const homeUrl = service.homeUrl;
+
+          // then
+          const expectedDefinedHomeUrl = `${service.definedHomeUrl}?lang=${this.intl.t('current-lang')}`;
+          expect(homeUrl).to.equal(expectedDefinedHomeUrl);
+        });
+      });
     });
 
-    it('should get "pix.fr" url when current domain contains pix.fr', function() {
-      // given
-      const service = this.owner.lookup('service:url');
-      const expectedHomeUrl = 'https://pix.fr';
-      service.definedHomeUrl = undefined;
-      service.currentDomain = { getExtension: sinon.stub().returns('fr') };
+    context('when it is not a Review App and environnement is ‘production‘', function() {
+
+      const defaultEnvironment = ENV.environment;
+      const defaultIsReviewApp = ENV.APP.REVIEW_APP;
+      let defaultLocale;
+
+      beforeEach(function() {
+        ENV.environment = 'production';
+        ENV.APP.REVIEW_APP = 'false';
+        defaultLocale = this.intl.t('current-lang');
+      });
+
+      afterEach(function() {
+        ENV.environment = defaultEnvironment;
+        ENV.APP.REVIEW_APP = defaultIsReviewApp;
+        this.intl.setLocale(defaultLocale);
+      });
+
+      [
+        { language: 'fr', currentDomainExtension: 'fr', expectedHomeUrl: 'https://pix.fr' },
+        { language: 'fr', currentDomainExtension: 'org', expectedHomeUrl: 'https://pix.org' },
+        { language: 'en', currentDomainExtension: 'fr', expectedHomeUrl: 'https://pix.fr/en-gb' },
```

Comment:
C'est un cas étrange qu'on voudrait éviter je pense.
Peut être qu'il faudrait qu'on est 
```suggestion
        { language: 'en', currentDomainExtension: 'fr', expectedHomeUrl: 'https://pix.org/en-gb' },
```

A vérifier peut être auprès des PO. Je ne pense pas que ce soit bloquant pour le moment.